### DISCUSSION TITLE
rabbitmq does not work in IPv6 environments

### DISCUSSION THREAD

**Discussion by lanzhiwang** [rabbitmq/cluster-operator#1226] [Q&A]
I want to verify that rabbimq works properly on ipv6 network protocol stack.

@lukebakken EDIT - removed contents because this question has been asked here: https://github.com/rabbitmq/rabbitmq-server/discussions/6820

**Comment by lukebakken** [comment]
Please don't post the same question in multiple locations

https://github.com/rabbitmq/rabbitmq-server/discussions/6820

**Comment by simonwu-sn** [comment]
https://github.com/rabbitmq/cluster-operator/pull/1455 raise a PR to fix the issue.
